Testscripts/Windows/VERIFY-DPDK-BUILD-AND-TESTPMD-TEST.ps1

```diff
@@ -10,6 +10,13 @@ function Main {
 	$currentTestResult = Create-TestResultObject
 
 	try {
+		Write-LogInfo "Restart VMs to restore them into original status."
```

Comment:
why we are restarting VMs?
I didn't get why we need to restart at the start of the test.

Reply:
when two cases run in sequence, after previous case run,  the eth1 is down. the following case will be affected.

Reply:
Yes, that might be happening because with Netvsc pmd setup we bind eth1 device to uio_hv_generic driver and unbind it from hv_netvsc driver. We need to restore settings back after the test is completed. This can only be achieved if we store DEV_UUID  and reverse steps in [Netvsc_device_setup()](https://github.com/microsoft/lisa/blob/2f0e3aaee8dde0dee67a27c8ca84ea79291a1ec1/Testscripts/Linux/dpdkUtils.sh#L731) function.

Reply:
@abhimarathe 
I revert change for this file. Please review this PR again.
Could you please submit a PR to fix this issue (eth1 is missing after some case)

Reply:
Sure. Thank you Lili!